frontend/app/src/filters.ts

```diff
@@ -51,17 +51,12 @@ export function balanceSum(value: BigNumber[]): BigNumber {
 export function aggregateTotal(
   balances: any[],
   mainCurrency: string,
-  exchangeRate: BigNumber,
-  precision: number
+  exchangeRate: BigNumber
 ): BigNumber {
   return balances.reduce((previousValue, currentValue) => {
     if (currentValue.asset === mainCurrency) {
-      return previousValue
-        .plus(currentValue.amount)
-        .dp(precision, BigNumber.ROUND_DOWN);
```

Comment:
Now that I look at it id the precision doesn't make really make sense anymore indeed.

It only made sense before AmountDisplay was introduced where the conversion was not at the UI level.

That is a good catch :)